daemon/execdriver/native/driver.go

```diff
@@ -295,18 +305,21 @@ func (d *driver) Terminate(c *execdriver.Command) error {
 	return err
 }
 
-func (d *driver) Info(id string) execdriver.Info {
+// Info returns information of native driver
```

Comment:
Info returns the configuration stored in the driver struct.
This is required to implement the execdriver `Driver` interface